.github/workflows/extract-translation-source-files.yml

```diff
@@ -34,36 +152,16 @@ jobs:
         git push --set-upstream origin ${{ steps.dynamic_branch.outputs.branch }}
 
   python-translations:
+    if: ${{ fromJson(needs.setup-matrix.outputs.has_python_repos) == true }}
```

Comment:
Is it possible/clean to check whether `python_repos` is nonempty instead of having a separate `has_python_repos` ?

I wouldn't be surprised if this is one of those things that GHA makes difficult, so def not a blocker.

Reply:
I definitely tried to go down that route and ran into issues (hence the `has_` vars). I'd love to refactor to that if I can figure out how to get GH actions to do what in js would be `python_repos.length > 0`, maybe you have had/will have better luck looking for docs/prior art on it?

Reply:
Sorry, I don't have any better ideas. Since you've already tried, I wouldn't worry about it 👍🏻